packages/flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -50,6 +52,7 @@ class SentryFlutterPlugin :
     pluginRegistrationTime = System.currentTimeMillis()
 
     context = flutterPluginBinding.applicationContext
+    applicationContext = context
```

Comment:
I'll replace the usage of `context` with the static `applicationContext` in a follow up PR